### PR TITLE
Add support for assets other than job_template (project, inventory, ...)

### PR DESCRIPTION
**Pull request by fpob**
* check `project` and `inventory` assets in `synchronize` command
* add command to download all assets including dependencies (`project` and `inventory`)

### CODE REVIEW

**Review by europ — APPROVED** [review]
Checked, seems good to me.

**Review by europ — CHANGES_REQUESTED** [review]
Rechecked, I missed out that `atacac/download.py` does not have unit tests, same for function `tower_list_all` in `atacac/_utils.py`.

File: atacac/_utils.py [hunk 1]
```diff
@@ -126,3 +162,12 @@ def load_asset(file_path):
...
         raise Error(f"Failed to read content of '{file_path}'")
+
+
+def sanitize_filename(filename):
+    """
+    Sanitize filename so it contains only alphanumeric characters, dot,
+    underscore or dash. All other characters are replaced with single
+    underscore.
+    """
+    return re.sub(r'[^a-zA-Z0-9.-]+', '_', filename)
```
Comment (europ) [inline atacac/_utils.py:174]: > All other characters are replaced with single underscore.

It is **false**, in some cases they are just excluded (not replaced with `_`).

```
Python 3.7.7 (default, Mar 13 2020, 21:39:43) 
[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import re
>>> filename = "a/b.c_d-e"
>>> re.sub(r'[^a-zA-Z0-9.-]+', '_', filename)
'a_b.c_d-e'
>>> filename = "a/_b"
>>> re.sub(r'[^a-zA-Z0-9.-]+', '_', filename)
'a_b'
>>> filename = "a#_b"
>>> re.sub(r'[^a-zA-Z0-9.-]+', '_', filename)
'a_b'
>>> 
```

**Review by europ — COMMENTED** [review]
Please check out my comment. Regardless of this little note, I would approve this PR.

File: atacac/_utils.py (hunk 1 above)
Comment (fpob) [reply atacac/_utils.py:174]: It is expected, consecutive underscores are collapsed into one eg. `a_%$#@_b` should be transformed to `a_b`. I updated doc.

**Review by europ — APPROVED** [review]
LGTM

**Pull request merged**